docs/index.rst

```diff
@@ -19,52 +19,50 @@ user-defined types among other features.
 With Solidity you can create contracts for uses such as voting, crowdfunding, blind auctions,
 and multi-signature wallets.
 
-.. note::
-    The best way to try out Solidity right now is using
-    `Remix <https://remix.ethereum.org/>`_
-    (it can take a while to load, please be patient). Remix is a web browser
-    based IDE that allows you to write Solidity smart contracts, then deploy
-    and run the smart contracts.
+Language Documentation
+----------------------
+
+If you are new to the concept of smart contracts we recommend you start with
+:ref:`an example smart contract <simple-smart-contract>` written
+in Solidity. When you are ready for more detail, we recommend you read the
```

Comment:
In all honestly, why would we list the sections in a different order here than in the TOC? If this is the recommended reading order, we should also change the TOC.

Reply:
I think it's perfectly fine to have an order that doesn't match the TOC, as this is specifically an introductory paragraph. But yes, we should also change the TOC anyway for other reasons too, as we are discussing in https://github.com/ethereum/solidity/issues/5151